package.json

```diff
@@ -163,7 +163,7 @@
     "react-dom": "^17.0.1",
     "react-g-analytics": "0.4.2",
     "react-router-dom": "^5.2.0",
-    "react-tiny-popover": "5",
+    "react-tiny-popover": "^6.0.1",
```

Comment:
We should consider getting rid of it as it doesn't work well on mobile:

<img src='https://user-images.githubusercontent.com/1091472/101637071-362ea080-3a67-11eb-9892-afa1fb5c68eb.jpeg' width='300' />

Reply:
What do you suggest?

Reply:
I'm considering remove the whole block, i.e., https://webpack.js.org/configuration/#options. It's almost not maintainable as far as I see, and it can become outdated quickly because of no maintenance.

Reply:
oh no, we need it, it functions as a single place to quickly explain all the options